terraform/service_redirect.tf

```diff
@@ -69,6 +137,8 @@ resource "google_cloud_run_service" "enx-redirect" {
           for_each = merge(
             local.gcp_config,
             local.enx_redirect_config,
+            local.cache_config,
+            local.database_config,
```

Comment:
I think you nee csrf and some other things too?

Reply:
@icco Can you comment here? I don't want to mess this up, like I did jwks, and I am **NOT** the person you want to leave as a babysitter on this one. :)

Reply:
Looking at cmd/enx-redirect/main.go I don't think you should need any other configs.